apps/web/src/atoms/first-load.ts

```diff
@@ -1,5 +1,47 @@
 import { atom } from 'jotai';
 import { atomWithStorage } from 'jotai/utils';
 
-export const isFirstLoadAtom = atomWithStorage<boolean>('isFirstLoad', true);
-export const openTipsAtom = atom<boolean>(false);
+export type FirstLoad = {
+  version: string;
+  openTips: boolean;
+  showWhatsNew: boolean;
+};
+
+export const isFirstLoadAtom = atomWithStorage<FirstLoad>('isFirstLoad', {
+  version: '0.0.0',
+  openTips: false,
+  showWhatsNew: false,
+});
```

Comment:
I think you should split them to each single atom

Reply:
Indeed, but I feel like there might be a way to unify storage? something like this.
```
export type FirstLoadGuide = {
  version: string;
  closedInCurrentSession: Record<string, boolean>;
  permanentlyClosed: Record<string, boolean>;
};

export const firstLoadAtom = atomWithStorage<FirstLoadGuide>('isFirstLoad', {
  version: CURRENT_APP_VERSION,
  closedInCurrentSession: {},
  permanentlyClosed: {},
});
```

Reply:
I won't work I believe. I think you should hide all tips by default and only show then in the second render. Not showing all of them in first time